biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -449,15 +453,21 @@ subroutine get_harvest_rate_area (patch_land_use_label, hlm_harvest_catnames, hl
      !  Normalize by site-level primary or secondary forest fraction
      !  since harvest_rate is specified as a fraction of the gridcell
      !  also need to put a cap so as not to harvest more primary or secondary area than there is in a gridcell
+     !  For secondary, also need to normalize by the young/old fraction.
      if (patch_land_use_label .eq. primaryland) then
         if (frac_site_primary .gt. fates_tiny) then
-           harvest_rate = min((harvest_rate / frac_site_primary),frac_site_primary)
+           harvest_rate = min((harvest_rate / frac_site_primary),1._r8)
         else
            harvest_rate = 0._r8
         endif
      else if (patch_land_use_label .eq. secondaryland) then
-        if (frac_site_secondary .gt. fates_tiny) then
-           harvest_rate = min((harvest_rate / frac_site_secondary), frac_site_secondary)
+        ! the .gt. -0.5 in the next line is because frac_site_secondary returns -1 if no secondary area.
+        if (frac_site_secondary .gt. fates_tiny .and. frac_site_secondary .gt. -0.5_r8) then
```

Comment:
I'm not pretty sure what does -0.5 mean here?

Reply:
really just checking that it is greater than -1, since that is the value returned if there is secondary area is less than fates_tiny.

Reply:
I see. Makes sense.

Reply:
@ckoven , I didn't see fates_tiny defined.  What is the its intent here?  Is that a generic very small number used to prevent div0s? Or is it the same as fortran tiny, ie the smallest possible number for this floating point type?  I very often use nearzero to prevent div0s

Reply:
yes, sorry, `nearzero`